project/Dependencies.scala

```diff
@@ -17,16 +17,16 @@ object Dependencies {
   val munitCatsEffectV = "1.0.7"
   val pact4sV = "0.9.0"
 
-  private val workbenchLibsHash = "63d2d78"
-  val serviceTestV = s"3.1-$workbenchLibsHash"
-  val workbenchModelV = s"0.18-$workbenchLibsHash"
+  private val workbenchLibsHash = "f87cad5"
```

Comment:
This should the latest and greatest wb-libs versions across all modules (except `google` which is marked with a TODO).

Reply:
Ohh, I know what happened! I recently updated workbench lib, and then leo, but forgot to update this hash 🤦‍♀️ Sorry about that, and thanks for fixing it